Thrive.csproj

```diff
@@ -305,7 +309,6 @@
     <Folder Include="shaders\" />
     <Folder Include="src\microbe_stage\organelle_components\" />
     <Folder Include="src\auto-evo\steps\" />
-    <Folder Include="src\auto-evo\simulation\" />
```

Comment:
I have no idea. This is an auto-generated file: I ran build, it chose to re-write things this way, and the code still runs. I'm certainly not going to dive into a csproj file and pretend I know better than the compiler.

Reply:
Could you then reset this change? If there's no reason for it.

Reply:
Anything I change by hand in this file should be overwritten by the next VS build, and I'm not sure there's no reason for it. It's an auto-generated file.

Reply:
It might be auto-generated but Rider will act differently depending on which folders are listed here, so I've added many of these through Rider manually.

Reply:
From the file diff, I don't see it added back  in the `Folder` `ItemGroup`. Am I missing something?